<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceWidget</class>
 <widget class="QWidget" name="AttendanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>24</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>汤臣二品物业考勤管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>45</height>
        </size>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>日常考核</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidgetdaily">
           <column>
            <property name="text">
             <string>考勤单编号</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>员工ID</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>员工姓名</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>部门</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>岗位</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>状态</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>年份：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxyear"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>月份：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxmonth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string> 日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxday">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonadd">
             <property name="text">
              <string>新增</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonalter">
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>月度汇总</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSrch">
               <property name="text">
                <string>查找</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>时间：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxtime"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonId">
               <property name="text">
                <string>按员工ID查询</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonName">
               <property name="text">
                <string>按员工姓名查询</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDpt">
               <property name="text">
                <string> 按部门查询</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonPst">
               <property name="text">
                <string>按岗位查询</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonStu">
               <property name="text">
                <string>按考勤状态查询</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidgetmonthly">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;黑体&quot;;
QTabBar::tab{height: 50;}</string>
           </property>
           <column>
            <property name="text">
             <string>月度考勤单编号</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>员工ID</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>员工姓名</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>部门</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>岗位</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>月份</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>是否全勤</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>请假次数</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>旷工次数</string>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
